### PR TITLE
Restyled app

### PR DESCRIPTION
**Pull request by mvkv**

### CODE REVIEW

File: flask_server.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def get_system_logs():
     log_file_path = settings.get('system_log_file_path')
     if not os.path.exists(log_file_path): return []
     with open(log_file_path, 'r+') as log_file:
-        return [log for log in log_file.readlines()]
+        return list(reversed([log for log in log_file.readlines()]))
```
Comment (powahftw) [inline flask_server.py]: [...].reverse() ?

File: flask_server.py [hunk 2]
```diff
@@ -98,7 +98,8 @@ def settings_page():
 @app.route("/gallery/<path:text>", methods=['GET'])
 def gallery(text):
     logger.info(f"GET request to /gallery/")
-    return render_template("gallery.html", title=text if text else "Gallery")
+    title = f"Gallery/{text}"
```
Comment (powahftw) [inline flask_server.py]: "Gallery" + f"/{text}" if ... else "" and remove the condition from line 102?

**Review by powahftw — APPROVED** [review]
Couple of suggestions

File: flask_server.py (hunk 1 above)
Comment (mvkv) [reply flask_server.py]: I think it's not working with list comprehension

File: flask_server.py (hunk 2 above)
Comment (mvkv) [reply flask_server.py]: Done

**Pull request merged**